index.html

```diff
@@ -98,55 +98,55 @@ <h2 class="article__title">SIMPLE AND BEAUTIFUL</h2>
 
     <section class="pricing" id="pricing">
 
-        <div class="pricing__title--container">
-            <h2 class="pricing__title">CHOOSE YOUR PRICING PLAN</h2>
+        <header class="pricing__title--container">
+            <h2 class="pricing__title title">CHOOSE YOUR PRICING PLAN</h2>
             <p class="pricing__subtitle">Pick any of our super affordable pricing plans</p>
-        </div>
+        </header>
 
         <div class="pricing__container">
-            <!-- PONIZSZE ELEMENTY ALTERNATYWNIE DO ZROBIENIA ZA POMOCĄ UL I LI -->
-            <div class="pricing__plan--left plan">
+            
+            <ul class="pricing__plan--left plan">
                 <h3 class="plan__title">BASIC</h3>
-                <p class="plan__price">$15</p>
-                <p class="plan__description--smaller">per month</p>
-                <p class="plan__description">Up to 7 Projects</p>
-                <p class="plan__description">2 Additional Developers</p>
+                <li class="plan__price">$15</li>
+                <li class="plan__description--smaller">per month</li>
+                <li class="plan__description">Up to 7 Projects</li>
+                <li class="plan__description">2 Additional Developers</li>
```

Comment:
Tylko te 2 elementy powinny być listą - reszta tj. nazwa pakietu czy cenna itp. już nie. To nie jest cześć listy.